xml/System.Data.Odbc/OdbcParameter.xml

```diff
@@ -746,9 +746,18 @@ public void CreateOdbcParameter()
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the offset of the data within the column.</summary>
+        <value>The offset of the data within the column. The default value is zero.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+ The <xref:System.Data.Odbc.OdbcParameter.Offset%2A> property is used for binary and string types. For fixed length data types, the value of <xref:System.Data.Odbc.OdbcParameter.Size%2A> is ignored.
```

Comment:
The build had this warning:

```
api/System.Data.Odbc.OdbcParameter.Offset.yml 
[Warning] 1 invalid cross reference(s) "<xref:System.Data.O.DbParameter.Size%2A>".
```
I think it's because there is no "Size" group:
https://review.docs.microsoft.com/en-us/dotnet/api/system.data.odbc.odbcparameter.size?view=netcore-3.0&branch=pr-en-us-3441

So my suggestion is to remove the `%2A` from the Size xrefs and see how the build goes:

```suggestion
 The <xref:System.Data.Odbc.OdbcParameter.Offset%2A> property is used for binary and string types. For fixed length data types, the value of <xref:System.Data.Odbc.OdbcParameter.Size> is ignored.
```

@roji can you please take a look at my suggestions and commit them if you agree?

Reply:
Sure thing. I added the %2A only because it's present everywhere - I have no idea what it actually does :)